BranchMetrics/ios-branch-deep-linking-attribution
# Release Automation

## Reference
SDK-1762 -- Release Automation
https://branch.atlassian.net/browse/SDK-1762

## Summary
Added following workflows -
* Pre Release SDK Integration Tests
* Post Release SDK Integration Tests
* Build and Run --> This workflow will update version and will create a Release Draft.At last it will also triggere a workflow in spm repo to create a release there.

## Type Of Change
- [ X] New feature (non-breaking change which adds functionality)

## Testing Instruction

Test Pre/Post Release SDK Integration Tests -
* Right now pre and post release QA GitHub Actions run on push.
* Create a fake / test commit on the test(/current) branch. Push code.
* Go to Github and open Actions tab to check new workflows are running and successful.

Test Build & Run Workflow
* This workflow is manually triggered. So go to Actions tab and manually trigger this workflow.
* Be careful in selecting the branch. This workflow will update version on current branch and then will Create a release draft.
* verify version is updated everywhere and release draft is created. Check if import action is triggered in spm repo.









cc @BranchMetrics/saas-sdk-devs for visibility.
